cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js

```diff
@@ -10,6 +10,9 @@ window.renderCloudinaryGalleryWidget = function () {
     if (cloudinaryObj) {
         if (cloudinaryObj.galleryEnabled && typeof cloudinary !== 'undefined') {
             var galleryOptions = cloudinaryObj.images.galleryWidget.options;
+            if (cloudinaryObj.basePath.indexOf('res.cloudinary.com') === -1) {
```

Comment:
the basePath is a URL and we need the domain.
Please get the domain out of the basePath, and do not use indexOf.
The domain should != 'res.cloudinary.com'